cvdupdate/__main__.py

```diff
@@ -200,16 +200,17 @@ def clean_all(config: str, verbose: bool):
 @cli.command("serve")
 @click.option("--config", "-c", type=click.Path(), required=False, default="", help="Config path. [optional]")
 @click.option("--verbose", "-V", is_flag=True, default=False, help="Verbose output. [optional]")
+@click.option("--update-interval-seconds", "-u",type=click.INT, required=False, default=0, help="Time in seconds before the next database update")
```

Comment:
- Shorthand is usually in capital (`-U`)
- (OCD) Missing space before `type=click.INT`